CleverTapSDK/InApps/CTInAppEvaluationManager.m

```diff
@@ -215,6 +279,25 @@ - (void)removeSentSuppressedClientSideInApps:(NSDictionary *)header {
     }
 }
 
+- (void)removeSentEvaluatedServerSideInAppIdsForProfile:(NSDictionary *)header {
+    NSArray *inapps_eval = header[CLTAP_INAPP_SS_EVAL_STORAGE_KEY_PROFILE];
+    if (inapps_eval && [inapps_eval count] > 0) {
+        NSUInteger len = inapps_eval.count > self.evaluatedServerSideInAppIdsForProfile.count ?  self.evaluatedServerSideInAppIdsForProfile.count : inapps_eval.count;
+        [self.evaluatedServerSideInAppIdsForProfile removeObjectsInRange:NSMakeRange(0, len)];
+        [self saveEvaluatedServerSideInAppIdsForProfile];
+    }
+}
+
+- (void)removeSentSuppressedClientSideInAppsForProfile:(NSDictionary *)header {
+    NSArray *suppresed_inapps = header[CLTAP_INAPP_SUPPRESSED_STORAGE_KEY_PROFILE];
```

Comment:
The header key is the same for profile and event queues, it should be `CLTAP_INAPP_SUPPRESSED_META_KEY`.